test/functional/test_framework/test_framework.py

```diff
@@ -464,7 +464,7 @@ def cache_path(n, *paths):
 
             for i in range(MAX_NODES):
                 for entry in os.listdir(cache_path(i)):
-                    if entry not in ['wallets', 'chainstate', 'blocks']:
+                    if entry not in ['wallets', 'chainstate', 'blocks', 'evodb', 'llmq']:
```

Comment:
Ooops, missed one more thing in my previous suggestion
```suggestion
                    if entry not in ['wallets', 'chainstate', 'blocks', 'evodb', 'llmq', 'backups']:
```